soh/soh/Enhancements/debugger/ImGuiHelpers.cpp

```diff
@@ -1,10 +1,11 @@
 #include "ImGuiHelpers.h"
+#include "../../../../libultraship/libultraship/ImGuiImpl.h"
 
 // Adds a text tooltip for the previous ImGui item
 void SetLastItemHoverText(const std::string& text) {
     if (ImGui::IsItemHovered()) {
         ImGui::BeginTooltip();
-        ImGui::Text(text.c_str());
+        ImGui::Text(SohImGui::BreakTooltip(text.c_str()).c_str());
```

Comment:
if you change the function to take a const std::string ref, then you won't have to call `text.c_str()` which might be a little cleaner.